lang/src/main/antlr4/org/kaazing/k3po/lang/parser/v2/Robot.g4

```diff
@@ -338,6 +338,10 @@ OptionKeyword: 'option';
 
 MaskKeyword: 'mask';
 
+TransportKeyword
+    : 'transport'
+    ;
+
```

Comment:
grammar is updated. location is adding some complexity since it is also expression, so the valuation needs to be delayed.